sasdata/fair_database/data/models.py

```diff
@@ -87,6 +87,10 @@ class Quantity(models.Model):
 
```

Comment:
I saw your comment/question about how deleting an OperationTree should work. I think cascade delete would handle this automatically if OperationTree had Quantity as an optional foreign key with `on_delete=models.CASCADE`, and the same for the foreign key to the parent operation.